### PR TITLE
feat(tooltip): add uib- prefix

### PR DESCRIPTION
**Pull request by wesleycho**
Note that this does it for the tooltip and popover (couldn't get both in the commit message due to conventional-changelog).

### CODE REVIEW

File: src/popover/popover.js [hunk 1]
```diff
@@ -15,36 +14,108 @@ angular.module( 'ui.bootstrap.popover', ['ui.bootstrap.tooltip'])
...
+    }
+  };
+}])
+
+.directive('popoverTemplate', ['$log', '$popoverSuppressWarning', '$uibTooltip', function($log, $popoverSuppressWarning, $tooltip) {
+  if (!$popoverSuppressWarning) {
+    $log.warn('popover-template is now deprecated. Use uib-popover-template instead.');
+  }
+
+  return $tooltip('popoverTemplate', 'popover', 'click', {
```
Comment (Foxandxss) [inline src/popover/popover.js]: I think I would make this use the new tooltip.

**Comment by Foxandxss** [comment]
Check my Modal PR to see how I managed to get all the deprecation messages out in tests.(you have lots of directives but few deprecation message tests).

File: src/popover/popover.js (hunk 1 above)
Comment (wesleycho) [inline src/popover/popover.js]: This is the old `popoverTemplate` directive

**Comment by wesleycho** [comment]
I think I just forgot to write the other tests

**Comment by wesleycho** [comment]
This PR should be good to review again.

**Comment by Foxandxss** [comment]
Missing doc updates, but I fixed that for you.

**Pull request closed without merging**